pkg/cli/app.go

```diff
@@ -145,20 +145,8 @@ func (a *App) Run() error {
 		),
 	})
 
-	ctx, cancel := context.WithCancel(context.Background())
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Can you change it as follows?
We use the log for monitoring and debugging.

```suggestion
	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
        defer cancel()
        
        go func() {
          <-ctx.Done()
          if err := ctx.Err(); err != nil {
            logger.Info("App is stopping due to signal (SIGINT or SIGTERM)")
          }
        }()
```